Project.toml

```diff
@@ -24,7 +24,7 @@ FerriteBlockArrays = "BlockArrays"
 FerriteMetis = "Metis"
 
 [compat]
-BlockArrays = "0.16"
+BlockArrays = "0.16, 1"
```

Comment:
How do you handle backwards compat?

Reply:
I believe there were no  breaking changes for the functionality used in the extension, just in the docs.

Reply:
No need, since the only changes required where in docs.
At least I couldn't find any use in ext/FerriteBlockArrays.jl relying on any breaking changes, the only thing was the rename of PseudoBlockArrays AFAIU which was only used in docs.